sdk/core/core-amqp/README.md

```diff
@@ -41,12 +41,12 @@ This SDK houses core AMQP common related functionality in use bu Azure SDKs that
 
 ## Examples
 
-Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.5/sdk/core/amqp-common/samples) directory for detailed samples.
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/core-amqp_1.0.0-preview.1/sdk/core/core-amqp/samples) directory for detailed samples.
```

Comment:
The updated link here won't work. The previous link used a "tag", we should use the "master" branch instead

Reply:
Updated!!